src/wasm-pack.nix

```diff
@@ -36,14 +36,14 @@ in
         or ''
           HOME=$(mktemp -d)
 
-          wasm-pack build \
+          wasm-pack build ${if args.workspace then args.pname else "./"} \
```

Comment:
This may cause some issues with workspaces where crates are defined within some nested path like `crates/*`. I think it's reasonable to just define a `cratePath` argument that defaults to `.`